Sources/Extensions/OptimizelyClient+Extension.swift

```diff
@@ -51,32 +51,34 @@ extension OptimizelyClient {
     ///   - sdkKey: sdk key
     ///   - logger: custom Logger
     ///   - eventDispatcher: custom EventDispatcher (optional)
+    ///   - datafileHandler: custom datafile handler (optional)
     ///   - userProfileService: custom UserProfileService (optional)
-    ///   - periodicDownloadInterval: custom interval for periodic background datafile download.
+    ///   - periodicDownloadInterval: interval in secs for periodic background datafile download.
     ///         The recommended value is 10 * 60 secs (you can also set this to nil to use the recommended value).
     ///         Set this to 0 to disable periodic downloading.
     ///   - defaultLogLevel: default log level (optional. default = .info)
     ///   - defaultDecisionOptions: default decision optiopns (optional)
     public convenience init(sdkKey: String,
                             logger: OPTLogger? = nil,
                             eventDispatcher: OPTEventDispatcher? = nil,
+                            datafileHandler: OPTDatafileHandler? = nil,
                             userProfileService: OPTUserProfileService? = nil,
                             periodicDownloadInterval: Int?,
```

Comment:
is this now required? since we are already allowing user to provide datafileHandler as a whole.

Reply:
@yasirfolio3 It should be much easier to turn on background polling in this way instead of extending the full datafileHandler.